crates/bevy_ecs/src/world/mod.rs

```diff
@@ -280,6 +281,30 @@ impl World {
             .unwrap_or_else(|| panic!("Entity {:?} does not exist", entity))
     }
 
+    /// Returns the components of an [`Entity`](crate::entity::Entity) through [`ComponentInfo`](crate::component::ComponentInfo).
+    #[inline]
+    pub fn inspect_entity(&mut self, entity: Entity) -> Vec<&ComponentInfo> {
```

Comment:
Should `self` be mutable? I tried removing it and tests still pass. I wonder why a warning is not emitted...

Reply:
It shouldn't be, as `&ComponentInfo` is not mutable. I just copy-pasted a different function and rewrote it, whoops. I'll get right on that.